MapboxCoreNavigation/RouteController.swift

```diff
@@ -428,6 +432,14 @@ open class RouteController: NSObject {
             outstandingFeedbackEvents.remove(at: index)
         }
     }
+    
+    /**
+     Set the rating and any comment the user may have about their route. Only used when exiting navigaiton.
+     */
+    @objc public func setEndOfRoute(rating: Int, comment: String?) {
```

Comment:
`setEndOfRoute(_:comment:)` is a weird name for a method that doesn’t set an `endOfRoute` property. How about `setRouteRating(_:comment:)` / `-setRouteRating:comment:` or `rateRoute(rating:comment:)` / `-rateRouteWithRating:comment:`?